ui/util/policy.js

```diff
@@ -9,6 +9,7 @@ import PropTypes from 'prop-types';
 
 export default class Policy {
   constructor(policy) {
+    PropTypes.checkPropTypes(Policy.propTypes, policy, 'property', 'Policy');
```

Comment:
Do we want to assert that the policy _parameter_ has this proptype or do we want to assert the _Policy_ has the proptype? Since we're using `Object.assign`, they equate to the same thing, they are semantically different, in my mind.